.github/workflows/lint-test-build-push.yaml

```
@@ -0,0 +1,207 @@
+name: Lint, Test, Build, Push
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+
+jobs:
+  lint-test:
```

It would probably be better to update this name to reflect the entire scope of the job:

```suggestion
  lint-test-build-push:
```